### PR TITLE
Update python3Packages.google*

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Update Google Python bindings.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by ruuda — COMMENTED** [review]
I haven’t verified whether the packages build or run, but the Nix changes look innocent enough, only version bumps and no other changes, so that looks good to me.

**Comment by SuperSandro2000** [comment]
Me sitting here and wondering why the hell I compile stdenv and then I see it: Why is this targeting staging?!

**Comment by fabaff** [comment]
> Why is this targeting staging?!

It felt that this would trigger a massive rebuild in the first place. Looks like that older PR are indeed targeting `master`. Thanks for switching the target branch.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 133298` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
18 packages marked as broken and skipped:
</summary>
<ul>
<li>python38Packages.arviz</li>
<li>python38Packages.dm-sonnet</li>
<li>python38Packages.graph_nets</li>
<li>python38Packages.optuna</li>
<li>python38Packages.pymc3</li>
<li>python38Packages.pywick</li>
<li>python38Packages.rl-coach</li>
<li>python38Packages.shap</li>
<li>python38Packages.skorch</li>
<li>python39Packages.arviz</li>
<li>python39Packages.dm-sonnet</li>
<li>python39Packages.graph_nets</li>
<li>python39Packages.optuna</li>
<li>python39Packages.pymc3</li>
<li>python39Packages.pywick</li>
<li>python39Packages.rl-coach</li>
<li>python39Packages.shap</li>
<li>python39Packages.skorch</li>
</ul>
</details>
<details>

  <summary>19 packages failed to build and already failed to build on hydra master:</summary>
  <ul>
<li>home-assistant: log was empty</li>
<li>python38Packages.gcsfs: log was empty</li>
<li>python38Packages.google-cloud-error-reporting: log was empty</li>
<li>python38Packages.google-cloud-logging: log was empty</li>
<li>python38Packages.pathy: log was empty</li>
<li>python38Packages.pytorch-metric-learning: log was empty</li>
<li>python38Packages.spacy: log was empty</li>
<li>python38Packages.spacy-transformers: log was empty</li>
<li>python38Packages.textacy: log was empty</li>
<li>python39Packages.gcsfs: log was empty</li>
<li>python39Packages.google-cloud-bigquery: log was empty</li>
<li>python39Packages.google-cloud-error-reporting: log was empty</li>
<li>python39Packages.google-cloud-logging: log was empty</li>
<li>python39Packages.pathy: log was empty</li>
<li>python39Packages.pytorch-metric-learning: log was empty</li>
<li>python39Packages.spacy: log was empty</li>
<li>python39Packages.spacy-transformers: log was empty</li>
<li>python39Packages.textacy: log was empty</li>
<li>s3ql: log was empty</li>
  </ul>
</details>
<details>

  <summary>21 packages failed to build and are new build failures:</summary>
  <ul>
<li>dvc-with-remotes: log was empty</li>
<li>python38Packages.baselines: log was empty</li>
<li>python38Packages.edward: log was empty</li>
<li>python38Packages.gensim: log was empty</li>
<li>python38Packages.google-cloud-automl: log was empty</li>
<li>python38Packages.google-cloud-bigquery: log was empty</li>
<li>python38Packages.google-cloud-resource-manager: log was empty</li>
<li>python38Packages.google-cloud-storage: log was empty</li>
<li>python38Packages.ignite: log was empty</li>
<li>python38Packages.mask-rcnn: log was empty</li>
<li>python38Packages.scikit-tda: log was empty</li>
<li>python38Packages.smart-open: log was empty</li>
<li>python38Packages.tflearn: log was empty</li>
<li>python38Packages.umap-learn: log was empty</li>
<li>python39Packages.gensim: log was empty</li>
<li>python39Packages.google-cloud-automl: log was empty</li>
<li>python39Packages.google-cloud-resource-manager: log was empty</li>
<li>python39Packages.google-cloud-storage: log was empty</li>
<li>python39Packages.ignite: log was empty</li>
<li>python39Packages.smart-open: log was empty</li>
<li>snakemake: log was empty</li>
  </ul>
</details>
<details>
<summary>
150 packages built:
</summary>
<ul>
<li>beancount (python39Packages.beancount)</li>
<li>deja-dup</li>
<li>duplicity</li>
<li>duply</li>
<li>fava</li>
<li>gcalcli</li>
<li>git-annex-remote-googledrive</li>
<li>gpt2tc</li>
<li>kmymoney</li>
<li>ledger2beancount</li>
<li>lieer</li>
<li>privacyidea</li>
<li>python38Packages.beancount</li>
<li>python38Packages.beancount_docverif</li>
<li>python38Packages.chirpstack-api</li>
<li>python38Packages.cirq</li>
<li>python38Packages.cirq-google</li>
<li>python38Packages.drivelib</li>
<li>python38Packages.goobook</li>
<li>python38Packages.google-api-core</li>
<li>python38Packages.google-api-python-client</li>
<li>python38Packages.google-auth</li>
<li>python38Packages.google-auth-httplib2</li>
<li>python38Packages.google-auth-oauthlib</li>
<li>python38Packages.google-cloud-access-context-manager</li>
<li>python38Packages.google-cloud-appengine-logging</li>
<li>python38Packages.google-cloud-asset</li>
<li>python38Packages.google-cloud-bigquery-datatransfer</li>
<li>python38Packages.google-cloud-bigquery-logging</li>
<li>python38Packages.google-cloud-bigtable</li>
<li>python38Packages.google-cloud-container</li>
<li>python38Packages.google-cloud-core</li>
<li>python38Packages.google-cloud-dataproc</li>
<li>python38Packages.google-cloud-datastore</li>
<li>python38Packages.google-cloud-dlp</li>
<li>python38Packages.google-cloud-dns</li>
<li>python38Packages.google-cloud-firestore</li>
<li>python38Packages.google-cloud-iam</li>
<li>python38Packages.google-cloud-iam-logging</li>
<li>python38Packages.google-cloud-iot</li>
<li>python38Packages.google-cloud-kms</li>
<li>python38Packages.google-cloud-language</li>
<li>python38Packages.google-cloud-monitoring</li>
<li>python38Packages.google-cloud-org-policy</li>
<li>python38Packages.google-cloud-os-config</li>
<li>python38Packages.google-cloud-pubsub</li>
<li>python38Packages.google-cloud-redis</li>
<li>python38Packages.google-cloud-runtimeconfig</li>
<li>python38Packages.google-cloud-secret-manager</li>
<li>python38Packages.google-cloud-securitycenter</li>
<li>python38Packages.google-cloud-spanner</li>
<li>python38Packages.google-cloud-speech</li>
<li>python38Packages.google-cloud-tasks</li>
<li>python38Packages.google-cloud-testutils</li>
<li>python38Packages.google-cloud-texttospeech</li>
<li>python38Packages.google-cloud-trace</li>
<li>python38Packages.google-cloud-translate</li>
<li>python38Packages.google-cloud-videointelligence</li>
<li>python38Packages.google-cloud-vision</li>
<li>python38Packages.google-cloud-websecurityscanner</li>
<li>python38Packages.google-resumable-media</li>
<li>python38Packages.gspread</li>
<li>python38Packages.kubernetes</li>
<li>python38Packages.oauthenticator</li>
<li>python38Packages.openshift</li>
<li>python38Packages.proto-plus</li>
<li>python38Packages.pydrive</li>
<li>python38Packages.pyro-ppl</li>
<li>python38Packages.pytorch (python38Packages.pytorchWithoutCuda)</li>
<li>python38Packages.pytorch-lightning</li>
<li>python38Packages.rising</li>
<li>python38Packages.slicer</li>
<li>python38Packages.tensorboardx</li>
<li>python38Packages.test-tube</li>
<li>python38Packages.torchgpipe</li>
<li>python38Packages.torchvision</li>
<li>python38Packages.weboob</li>
<li>python38Packages.woob</li>
<li>python39Packages.baselines</li>
<li>python39Packages.beancount_docverif</li>
<li>python39Packages.chirpstack-api</li>
<li>python39Packages.cirq</li>
<li>python39Packages.cirq-google</li>
<li>python39Packages.drivelib</li>
<li>python39Packages.edward</li>
<li>python39Packages.goobook</li>
<li>python39Packages.google-api-core</li>
<li>python39Packages.google-api-python-client</li>
<li>python39Packages.google-auth</li>
<li>python39Packages.google-auth-httplib2</li>
<li>python39Packages.google-auth-oauthlib</li>
<li>python39Packages.google-cloud-access-context-manager</li>
<li>python39Packages.google-cloud-appengine-logging</li>
<li>python39Packages.google-cloud-asset</li>
<li>python39Packages.google-cloud-bigquery-datatransfer</li>
<li>python39Packages.google-cloud-bigquery-logging</li>
<li>python39Packages.google-cloud-bigtable</li>
<li>python39Packages.google-cloud-container</li>
<li>python39Packages.google-cloud-core</li>
<li>python39Packages.google-cloud-dataproc</li>
<li>python39Packages.google-cloud-datastore</li>
<li>python39Packages.google-cloud-dlp</li>
<li>python39Packages.google-cloud-dns</li>
<li>python39Packages.google-cloud-firestore</li>
<li>python39Packages.google-cloud-iam</li>
<li>python39Packages.google-cloud-iam-logging</li>
<li>python39Packages.google-cloud-iot</li>
<li>python39Packages.google-cloud-kms</li>
<li>python39Packages.google-cloud-language</li>
<li>python39Packages.google-cloud-monitoring</li>
<li>python39Packages.google-cloud-org-policy</li>
<li>python39Packages.google-cloud-os-config</li>
<li>python39Packages.google-cloud-pubsub</li>
<li>python39Packages.google-cloud-redis</li>
<li>python39Packages.google-cloud-runtimeconfig</li>
<li>python39Packages.google-cloud-secret-manager</li>
<li>python39Packages.google-cloud-securitycenter</li>
<li>python39Packages.google-cloud-spanner</li>
<li>python39Packages.google-cloud-speech</li>
<li>python39Packages.google-cloud-tasks</li>
<li>python39Packages.google-cloud-testutils</li>
<li>python39Packages.google-cloud-texttospeech</li>
<li>python39Packages.google-cloud-trace</li>
<li>python39Packages.google-cloud-translate</li>
<li>python39Packages.google-cloud-videointelligence</li>
<li>python39Packages.google-cloud-vision</li>
<li>python39Packages.google-cloud-websecurityscanner</li>
<li>python39Packages.google-resumable-media</li>
<li>python39Packages.gspread</li>
<li>python39Packages.kubernetes</li>
<li>python39Packages.mask-rcnn</li>
<li>python39Packages.oauthenticator</li>
<li>python39Packages.openshift</li>
<li>python39Packages.proto-plus</li>
<li>python39Packages.pydrive</li>
<li>python39Packages.pyro-ppl</li>
<li>python39Packages.pytorch (python39Packages.pytorchWithoutCuda)</li>
<li>python39Packages.pytorch-lightning</li>
<li>python39Packages.rising</li>
<li>python39Packages.scikit-tda</li>
<li>python39Packages.slicer</li>
<li>python39Packages.tensorboardx</li>
<li>python39Packages.test-tube</li>
<li>python39Packages.tflearn</li>
<li>python39Packages.torchgpipe</li>
<li>python39Packages.torchvision</li>
<li>python39Packages.umap-learn</li>
<li>python39Packages.weboob</li>
<li>python39Packages.woob</li>
<li>tts</li>
</ul>
</details>
